spirv_cross.cpp

```diff
@@ -70,6 +70,12 @@ Compiler::Compiler(vector<uint32_t> ir)
 	parse();
 }
 
+Compiler::Compiler(const uint32_t* ir, size_t wordCount)
```

Comment:
Asterisk binds to the right (Clang format script in format_all.sh).
word_count instead of wordCount.